lib/tasks/vote.rake

```diff
@@ -0,0 +1,25 @@
+require 'ffaker'
+namespace :vote do
+  desc 'Vote for specific debate and answer'
```

Comment:
You should define description for tasks because now if we call `rails -T` it lacks info about tasks

```
...
rails tmp:create                           # Creates tmp directories for cache, sockets, and pids
rails vote:positive[debate_id,auth_token]  # Vote for specific debate and answer
```